packages/google_maps_flutter/google_maps_flutter/example/ios/Podfile

```diff
@@ -34,6 +34,9 @@ target 'Runner' do
 
     pod 'OCMock', '~> 3.9.1'
   end
+  target 'RunnerUITests' do
```

Comment:
I wanted to use `CLLocationCoordinate2D` in these tests so I changed this to allow importing `GoogleMaps`.